Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -24,3 +24,27 @@ extension Array where Element: NSTextCheckingResult {
         return map { $0.range }
     }
 }
+
+extension Array where Element: Equatable {
+    var unique: [Element] {
+        var uniqueValues: [Element] = []
+        forEach { item in
+            if !uniqueValues.contains(item) {
+                uniqueValues += [item]
+            }
+        }
+        return uniqueValues
+    }
+}
+
+extension SequenceType {
+    // swiftlint:disable line_length
```

Comment:
if you just want to disable for the next line (as opposed to the entire rest of the file, as is happening currently), change this to `// swiftlint:disable:next line_length`